Tests/fixtures/desktop.yml

```diff
@@ -6876,3 +6876,21 @@
     model: Mac Mini (2018)
   os_family: Mac
   browser_family: Unknown
+-
+  user_agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.83 AlohaBrowser/0.1.0 Safari/537.36
+  os:
+    name: Windows
+    version: "10"
+    platform: x64
+  client:
+    type: browser
+    name: Aloha Browser
+    version: "0.1.0"
+    engine: WebKit
```

Comment:
set default engine: Blink
need add new rule in file regexes/client/browsers.yml
```yml
- regex: 'Chrome/.+ AlohaBrowser(?:/(\d+[\.\d]+))?'
  name: 'Aloha Browser'
  version: '$1'
  engine:
    default: 'Blink'
- regex: 'AlohaBrowser(?:/(\d+[\.\d]+))?'
...    
```

Reply:
Done. 
Failing tests were fixed as well.